### DISCUSSION TITLE
How to upload files larger than 20MB

### DISCUSSION THREAD

**Discussion by chenxiaowei107** [Budibase/budibase#12796] [Help] [answered]
When I want upload files larger than 20MB。BudiBase popup a error :"Files cannot exceed 20MB. Please try again with smaller files."
I tried to find the answer on google, but it didn't work。Can anyone tell me how I should modify it?
![image](https://github.com/Budibase/budibase/assets/39343523/6b6d394d-f2b8-4a6d-9553-0b2960b84fa8)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @chenxiaowei107,

Currently there is no way to increase this cap. I'd honestly recommend compressing anything that even exceeds 10mbs. As this will drastically slow down your application.

**Reply by chenxiaowei107** [reply]
I am using it in an intranet network,I have so many files ,but the speed is acceptable。
I Think the program should have a option to set the biggest file。
![image](https://github.com/Budibase/budibase/assets/39343523/72fcfd91-7c76-41ac-83c0-5286a1f2ba27)
